CHANGELOG.md

```diff
@@ -27,12 +27,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
   - `TextLog`: `write`, `clear`
   - `TreeNode`: `expand`, `expand_all`, `collapse`, `collapse_all`, `toggle`, `toggle_all`
   - `Tree`: `clear`, `reset`
-- Replaced some private attributes with public ones in the json tree example. https://github.com/Textualize/textual/pull/2138
+- Screens with alpha in their background color will now blend with the background. https://github.com/Textualize/textual/pull/2139
```

Comment:
Was it intentional that this nuked the CHANGELOG entry about the JSON tree example?

Reply:
Yeah, examples aren't relevant. Folk only care about the API.